### PR TITLE
Remove support for `stack`

### LINKED ISSUE #113: Fix the stack build
I'm unable to build the examples from master on a stock Ubuntu VM using Stack. I'm following the instructions on the README.

Steps to reproduce:

1. Grab an Ubuntu VM. I've tried several versions, including 20.10 and 18.04 LTS.
1. Run 'sudo apt-get update`
1. Generate an SSH key and upload it to GitHub. This step is required because stack.yaml uses GitHub urls that start with "git://", which forces the use to authenticate. This would be simplified if stack.yaml used the https GitHub urls.
1. Install Stack following https://docs.haskellstack.org/en/stable/install_and_upgrade/. I also tried older versions of Stack to try to build older commits.
1. Install Nix following https://nixos.org/download.html
1. `git clone https://github.com/awakesecurity/gRPC-haskell.git`
1. Run `nix-shell release.nix -A grpc-haskell.env`, which succeeds.
1. Run `stack build --fast && stack test --fast`, which fails.

 Expected result: Build succeeds.

Actual result:

build fails with an error

```
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for grpc-haskell-0.0.2.0:
    proto3-suite-0.4.0.0 from stack configuration does not match >=0.4.2.0 
    proto3-wire-1.1.0 from stack configuration does not match >=1.2.0  (latest matching version is 1.2.0)
needed since grpc-haskell is a build target.

```
It is straightforward to fix `stack build` by upgrading the dependencies in stack.yaml. However, `stack test` and `stack build --flag grpc-haskell:with-examples` both fail. Both complain about missing dependencies.

I tried installing various versions of grpc in the ambient OS (outside of the nix shell), but that did not help. For example, I tried gRPC 1.2 and gRPC 1.22 with various git commits.

I'm unable to make the instructions as written work with any combination of Stack, Ubuntu, and gRPC or any git commit. My guess is that when the Stack build works it's because it's pulling extra dependencies from the developers' machines, and those dependencies aren't explicitly modeled in the build configs.

### PR DESCRIPTION
**Pull request by Gabriel439**
Fixes #113

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,108 +17,16 @@ There is a tutorial [here](examples/tutorial/TUTORIAL.md)
...
-If you want to use `stack` in a relatively natural and painless manner, you will
-need a working installation of the GRPC C core libraries.
-
-On MacOS, because of
-issues [related](https://github.com/commercialhaskell/stack/issues/1161) to
-System Integrity Protection, dependencies on the nix-built `grpc` don't seem to
-work properly in the stack toolflow when `DYLD_LIBRARY_PATH` refers to the
-`nix`-built `grpc` library in the nix store, so the library needs to be
-installed somewhere that the loader can pick it up without `DYLD_LIBRARY_PATH`
-set (e.g., in `/usr/local/lib`).
```
Comment (evanrelf) [inline README.md:67]: I think this note about `grpc` from Nixpkgs on macOS is still relevant, no?

Comment (Gabriel439) [reply README.md:67]: @evanrelf: Not at as far as I can tell.  I've been doing `gRPC-haskell` development on OS X without issue on `master`

Comment (evanrelf) [reply README.md:67]: Okay cool, that's good to hear

**Review by evanrelf — APPROVED** [review]
If you're able to work on `gRPC-haskell` on macOS using a Nixpkgs-provided `grpc` without issue, then LGTM

**Pull request merged**